routers/db_functions.py

```diff
@@ -0,0 +1,22 @@
+"""
+db_functions.py
```

Comment:
Those two functions look more like FastAPI dependencies. I think it would be more appropriate to move them from the routers to the dependencies.py module in the root folder of your project